### DISCUSSION TITLE
Zoom is not working

### DISCUSSION THREAD

**Discussion by projct1** [biati-digital/glightbox#493] [Q&A]
In opened lightbox double-tap or two-fingers tap and resize - zoom are not happened.

**Comment by gingerchew** [comment]
Can you provide an example? Something like a CodePen helps us to understand what is going on, much more helpful than a general description.

**Comment by projct1** [comment]
https://github.com/biati-digital/glightbox/assets/9752959/4438d3dc-46ba-4e7c-9213-fdf459045606

By default zooming is true, but its not working even with `{ zoomable: true }` or `data-zoomable="true"`

Look video.

**Comment by inkShio** [comment]
https://github.com/biati-digital/glightbox/blob/a9385e50828b90e65bfc146a7c7e196081ba1a0d/src/js/core/zoom.js#L57

This line strictly states that if <= 768 it will not work.
If you remove this condition and add these styles (which are also removed by the media query), then everything starts to work, BUT another problem appears.

<img width="688" height="197" alt="image" src="https://github.com/user-attachments/assets/0e8ace97-bbb6-4cd7-9266-6597fb49e203" />
